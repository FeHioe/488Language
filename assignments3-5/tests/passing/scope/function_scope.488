% test function scope
{
  % function scope
  function isEqual ( value1 : Integer, value2 : Integer ) : Boolean {
    write "value 1 is ", value1, newline
    write "value 2 is ", value2
    return with value1 = value2
  }
  % statement required in scope when there is declaration

  var x : Boolean
  x := isEqual(1, 1)
}
