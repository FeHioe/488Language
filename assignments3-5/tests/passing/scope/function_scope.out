value 1 is 1
value 2 is 1
